lib/restforce/db/record_types/salesforce.rb

```diff
@@ -23,14 +23,15 @@ def create!(from_record)
 
           # NOTE: #upsert! returns a String Salesforce ID when a record is
           # created, and returns `true` when an existing record was found.
-          if record_id.is_a?(String)
-            from_record.update!(@mapping.lookup_column => record_id).after_sync
```

Comment:
I was confused at the removal of `after_sync` here, until I saw that it's part of `update!`. Nice.